osf_tests/test_archiver.py

```diff
@@ -758,6 +758,7 @@ def test_handle_archive_fail(self, mock_send_mail):
             {}
         )
         assert_equal(mock_send_mail.call_count, 2)
+        self.dst.refresh_from_db()
```

Comment:
Comment: we use `reload()` and `refresh_from_db()` interchangeably, though they are the same, see [here](https://github.com/CenterForOpenScience/osf.io/blob/d07206d3a7b4a26e440c66131531627170e9b1c9/osf/models/base.py#L107-L108).